pages/services/spark/2.6.0-2.3.2/limitations/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Limitations of DC/OS Apache Spark
+title: Limitations
+menuWeight: 135
+featureMaturity:
+render: mustache
+model: /services/spark/data.yml
+---
+
+*   Mesosphere does not provide support for {{ model.techShortName }} app development, such as writing a Python app to process data from
+    Kafka or writing Scala code to process data from HDFS.
+
+*   {{ model.techShortName }} jobs run in Docker containers. The first time you run a {{ model.techShortName }} job on a node, it might take longer than you
+    expect because of the `docker pull`.
+
+*   DC/OS Apache {{ model.techShortName }} only supports running the {{ model.techShortName }} shell from within a DC/OS cluster. See the {{ model.techShortName }} Shell section
+    for more information. For interactive analytics, we recommend Zeppelin, which supports visualizations and dynamic
+    dependency management.
+
+*   With {{ model.techShortName }} SSL/TLS enabled, if you specify environment-based secrets with
+    `{{ model.serviceName }}.mesos.[driver|executor].secret.envkeys, the keystore and truststore secrets will also show up as
```

Comment:
Not your fault but would you mind fixing this by adding the missing backtick after `secret.envkeys` and also for pages `2.5.0, 2.4.0, 2.3.1, 2.3.0` too?